packages/logger/_test.pony

```diff
@@ -85,14 +90,14 @@ class _TestObjectLogging is _LoggerTest[U64]
 
   fun level(): LogLevel => Fine
 
-  fun tag expected(): String => "42\n"
+  fun tag expected(): String => "FINE: 42\n"
 
   fun log(logger': Logger[U64]) =>
     logger'(Error) and logger'.log(U64(42))
 
 primitive _TestFormatter is LogFormatter
```

Comment:
I don't think there is much value in these test changes. I'd prefer to see them remain as they were.

Reply:
Removed.